addons/mail/static/src/components/message/message_tests.js

```diff
@@ -337,18 +345,22 @@ QUnit.test('Notification Error', async function (assert) {
 QUnit.test("'channel_fetch' notification received is correctly handled", async function (assert) {
     assert.expect(3);
 
+    this.data['res.partner'].records.push({
+        id: 11,
+        display_name: "Recipient",
```

Comment:
since you have diff on the lines: alphabetical

Reply:
Done